src/applications/survivors-benefits/config/chapters/04-household-information/dependentsPages.js

```diff
@@ -0,0 +1,457 @@
+import React from 'react';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import {
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoUI,
+  arrayBuilderYesNoSchema,
+  fullNameUI,
+  fullNameSchema,
+  textUI,
+  textSchema,
+  yesNoUI,
+  yesNoSchema,
+  radioUI,
+  radioSchema,
+  ssnUI,
+  ssnSchema,
+  checkboxUI,
+  checkboxSchema,
+  currentOrPastDateUI,
+  currentOrPastDateSchema,
+  selectUI,
+  addressUI,
+  addressSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import {
+  STATE_NAMES,
+  STATE_VALUES,
+  COUNTRY_NAMES,
+  COUNTRY_VALUES,
+} from '../../../utils/labels';
+import {
+  DependentChildDescription,
+  seriouslyDisabledDescription,
+} from '../../../utils/helpers';
+import { VaForm214138Alert } from '../../../components/FormAlerts';
+
+/**
+ * Dependent children (array builder)
+ */
+
+const options = {
```

Comment:
The alertMax text is not correct. I have code in my chapter 6 updates you can pull in and update to make it match.